app/models/state_file_md_intake.rb

```diff
@@ -75,6 +79,14 @@
 class StateFileMdIntake < StateFileBaseIntake
   encrypts :account_number, :routing_number, :raw_direct_file_data, :raw_direct_file_intake_data
 
+  enum eligibility_lived_in_state: { unfilled: 0, yes: 1, no: 2 }, _prefix: :eligibility_lived_in_state
+  enum eligibility_out_of_state_income: { unfilled: 0, yes: 1, no: 2 }, _prefix: :eligibility_out_of_state_income
```

Comment:
we should use a consistent type for the `_prefix` parameter to `enum`. i know i suggested passing `true` because it has the same effect as passing the name of the enum & is less repetitive/DRYer, but it's also inconsistent with the rest of the codebase. for that reason, i think it's probably worth changing lines 84-87 to pass the name of the enum as the `_prefix` argument, as lines 82-83 do

Reply:
Good catch, updated to pass the enum name to prefix